src/Redux/Store.js

```diff
@@ -9,6 +11,9 @@ if (process.env.NODE_ENV === "development") {
   composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
 }
 
-const configureStore = () => createStore(rootReducer, composeEnhancers(applyMiddleware(...middlewares)));
+const configureStore = () => {
+  const preloadedState = process.env.REACT_APP_SANDBOX ? sandboxState : undefined;
```

Comment:
process.env variables would always be casted to Strings.. so this `might` be always true.

Reply:
Same as [comment](https://github.com/singnet/refactored-dapp/pull/110#discussion_r309243540)